### PR TITLE
test: improve reliability of flaky test

### PR DESCRIPTION
**Pull request by crleona**
### Summary

fixes flakey test

### Checklist

* [X] Does your PR title have the correct [title format](https://github.com/amplitude/Amplitude-SDK-Template/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:  no

### CODE REVIEW

**Review by izaaz — APPROVED** [review]

File: Tests/AmplitudeTests/Utilities/EventPipelineTests.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ import XCTest
 @testable import AmplitudeSwift
 
 final class EventPipelineTests: XCTestCase {
-    private static let FLUSH_INTERVAL_SECONDS = 1.0
+    private static let FLUSH_INTERVAL_SECONDS = 300.0
```
Comment (izaaz) [inline Tests/AmplitudeTests/Utilities/EventPipelineTests.swift:13]: Are there no tests that depend on this being 1 second?

Comment (crleona) [reply Tests/AmplitudeTests/Utilities/EventPipelineTests.swift:13]: There are none, but this isn't the proper fix - I can't reproduce the failures locally so I'm using this PR to debug.

**Review by yuhao900914 — COMMENTED** [review]
Should this be `test` instead of `fix`?

**Comment by crleona** [comment]
> Should this be `test` instead of `fix`?

yup - switched!

**Review by yuhao900914 — APPROVED** [review]

**Pull request merged**